client/rest/test/plugins/rosetta/accountRoutes_spec.js

```diff
@@ -199,10 +241,212 @@ describe('account routes', () => {
+			const transactionJson = createUnconfirmedTransactionsResponse([transaction]);
+
+			// Act + Assert:
+			await assertIncludeMempoolAccountTest(transactionJson, expectedCoins);
+		});
+
+		it('succeeds when multiple coins sent in the mempool (include fee)', async () => {
+			// Arrange:
+			const mosaicsToSend = [
+				{ mosaicId: generateMosaicAliasId('cat.dog'), amount: 100 },
+				{ mosaicId: generateMosaicAliasId('symbol.xym'), amount: 2 }
+			];
+			const expectedCoins = [
+				createRosettaCoin('123', 'foo.bar', 3, '1122334455667788'),
+				createRosettaCoin('162', 'cat.dog', 4, '0F61E4A360897965'),
+				createRosettaCoin('10', 'symbol.xym', 6, '1ABBCCDDAABBCCDD')
+			];
+
+			const transaction = createTransactionJson(mosaicsToSend, TARGET_SIGNER_PUBLIC_KEY, TARGET_RECIPIENT_ADDRESS);
+			const transactionJson = createUnconfirmedTransactionsResponse([transaction]);
+
+			// Act + Assert:
+			await assertIncludeMempoolAccountTest(transactionJson, expectedCoins);
+		});
+
+		it('succeeds when multiple transactions receive in the mempool', async () => {
+			// Arrange:
+			const mosaicsToSend = [
+				{ mosaicId: generateMosaicAliasId('cat.dog'), amount: 100 },
+				{ mosaicId: generateMosaicAliasId('symbol.xym'), amount: 2 }
+			];
+			const mosaicsToReceive = [
+				{ mosaicId: generateMosaicAliasId('cat.dog'), amount: 50 },
+				{ mosaicId: generateMosaicAliasId('symbol.xym'), amount: 20 },
+				{ mosaicId: generateMosaicAliasId('foo.bar'), amount: 23 }
+			];
+			const expectedCoins = [
+				createRosettaCoin('146', 'foo.bar', 3, '1122334455667788'),
+				createRosettaCoin('212', 'cat.dog', 4, '0F61E4A360897965'),
+				createRosettaCoin('30', 'symbol.xym', 6, '1ABBCCDDAABBCCDD')
+			];
+			const outgoingTransaction = createTransactionJson(mosaicsToSend, TARGET_SIGNER_PUBLIC_KEY, TARGET_RECIPIENT_ADDRESS);
+			const incomingTransaction = createTransactionJson(mosaicsToReceive, OTHER_SIGNER_PUBLIC_KEY, OTHER_RECIPIENT_ADDRESS);
+			const transactionJson = createUnconfirmedTransactionsResponse([outgoingTransaction, incomingTransaction]);
+
+			// Act + Assert:
+			await assertIncludeMempoolAccountTest(transactionJson, expectedCoins);
+		});
+
+		it('succeeds when include mempool is false', async () => {
+			// Arrange:
+			const mosaicsToSend = [
+				{ mosaicId: generateMosaicAliasId('cat.dog'), amount: 100 },
+				{ mosaicId: generateMosaicAliasId('symbol.xym'), amount: 2 }
+			];
+			const expectedCoins = [
+				createRosettaCoin('123', 'foo.bar', 3, '1122334455667788'),
+				createRosettaCoin('262', 'cat.dog', 4, '0F61E4A360897965'),
+				createRosettaCoin('112', 'symbol.xym', 6, '1ABBCCDDAABBCCDD')
+			];
+			const transaction = createTransactionJson(mosaicsToSend, TARGET_SIGNER_PUBLIC_KEY, TARGET_RECIPIENT_ADDRESS);
+			const transactionJson = createUnconfirmedTransactionsResponse([transaction]);
+
+			// Act + Assert:
+			await assertIncludeMempoolAccountTest(transactionJson, expectedCoins, false);
```

Comment:
if possible try to add test that parses aggregate

Reply:
ok.  I will take a look.  I guess there should be one also for the block routes